.github/workflows/rebuild-country.yml

```diff
@@ -71,13 +71,15 @@ jobs:
           echo
           echo "--> When completed, the following will be available:"
           echo "build files: s3://nextstrain-ncov-private/trial/$TRIAL_NAME/"
-          echo "nextstrain URLs: https://nextstrain.org/staging/ncov/gisaid/trial/$TRIAL_NAME/REGION_NAME"
+          echo "nextstrain URLs: https://nextstrain.org/staging/ncov/gisaid/trial/$TRIAL_NAME/REGION_NAME/TIME_SPAN"
+          echo "e.g. https://nextstrain.org/staging/ncov/gisaid/trial/$TRIAL_NAME/global/all-time"
```

Comment:
Not sure if `global` is appropriate for rebuild-country.yml.

Reply:
Hmm. I'm not really sure how the rebuild-country workflow introduced by @trvrb in #970 is supposed to work. I think there's a lot of copy pasta from rebuild-gisaid adding to the confusion (e.g. @jover fixed another instance in #972), such as what these URLs should be. Some digging reveals that:

The `nextstrain_profiles/nextstrain-country` profile sets a default `auspice_json_prefix`:

https://github.com/nextstrain/ncov/blob/3e767e9fa125800c4664dea753e48f227916310c/nextstrain_profiles/nextstrain-country/builds.yaml#L1

but the rebuild-country workflow overrides `auspice_json_prefix`:

https://github.com/nextstrain/ncov/blob/3e767e9fa125800c4664dea753e48f227916310c/.github/workflows/rebuild-country.yml#L44

if a trial name was given when the workflow was launched.

The profile also contains references to a `{country}` placeholder:

https://github.com/nextstrain/ncov/blob/3e767e9fa125800c4664dea753e48f227916310c/nextstrain_profiles/nextstrain-country/builds.yaml#L62-L141

which I believe takes its value from the `builds:` entries:

https://github.com/nextstrain/ncov/blob/3e767e9fa125800c4664dea753e48f227916310c/nextstrain_profiles/nextstrain-country/builds.yaml#L44-L56

So that means it's hardcoded in the profile but apparently not reflected in the output files or final Auspice dataset JSONs. The names of the latter then depend on how the workflow is invoked and may be either:

```
ncov_country_nextstrain_country_{2m,6m,all-time}*.json
ncov_gisaid_trial_*_nextstrain_country_{2m,6m,all-time}*.json
```

Furthermore, it's not clear how the nextstrain-country profile works as-is (at least with the current codebase), as I believe producing the `auspice_config.json` file (e.g. `results/nextstrain_country_all-time/auspice_config.json` where `nextstrain_country_all-time` is one of the `builds:` keys defined by the profile) will fail due to violated assumptions:

```
$ nextstrain build . results/nextstrain_country_all-time/auspice_config.json --profile nextstrain_profiles/nextstrain-country/
Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 8
Rules claiming more threads will be scaled down.
Job counts:
        count   jobs
        1       auspice_config
        1
Cannot send slack message as the config does not define a channel and/or token.

[Wed Aug 24 17:15:53 2022]
Job 0: Making a custom auspice config.
Reason: Missing output files: results/nextstrain_country_all-time/auspice_config.json

Job counts:
        count   jobs
        1       auspice_config
        1
[Wed Aug 24 17:15:56 2022]
Error in rule auspice_config:
    jobid: 0
    output: results/nextstrain_country_all-time/auspice_config.json

RuleException:
ValueError in line 146 of /nextstrain/build/workflow/snakemake_rules/export_for_nextstrain.smk:
too many values to unpack (expected 2)
  File "/nextstrain/build/workflow/snakemake_rules/export_for_nextstrain.smk", line 146, in __rule_auspice_config
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Exiting because a job execution failed. Look above for error message
…
```

I suppose this could be fixed by specifying a pre-made file in each build's params, which is supported by:

https://github.com/nextstrain/ncov/blob/3e767e9fa125800c4664dea753e48f227916310c/workflow/snakemake_rules/main_workflow.smk#L1378-L1389

or adjusting the `auspice_config` rule responsible for generating the file:

https://github.com/nextstrain/ncov/blob/3e767e9fa125800c4664dea753e48f227916310c/workflow/snakemake_rules/export_for_nextstrain.smk#L143-L149

All in all, my conclusion here is that the nextstrain-country profile and rebuild-country workflow were a quick one-off thing to support a timely need and need a bit more attention to detail **iff** they're going to continue to be maintained/supported parts of the repo or our processes.

For this PR, I believe the `REGION_NAME` placeholder and `global` example value should be replaced by `COUNTRY` and `nextstrain/country`.

Reply:
Trevor mentioned in [Slack](https://bedfordlab.slack.com/archives/CTZKJC7PZ/p1656715580308109?thread_ts=1656706685.515619&cid=CTZKJC7PZ) that `rebuild-country` was only intended to be manually triggered.